crates/alloy/Cargo.toml

```diff
@@ -72,6 +72,21 @@ std = [
     "alloy-consensus?/std",
 ]
 
+# full
+full = [
+    "consensus",
+    "contract",
+    "eips",
+    "k256",
+    "kzg",
```

Comment:
imo this makes sense to include in `full`

full will probably include more stuff than necessary but for convenience this is okay imo.